src/astronomicalObject/AstronomicalObject.ts

```diff
@@ -102,7 +104,7 @@ export default abstract class AstronomicalObject implements IAstronomicalObject
     public async getApparentTopocentricHorizontalCoordinates(
         location: Location
     ): Promise<LocalHorizontalCoordinates> {
-        const {azimuth, altitude, radiusVector} = await this.getTopocentricHorizontalCoordinates(location);
+        const { azimuth, altitude, radiusVector } = await this.getTopocentricHorizontalCoordinates(location);
```

Comment:
I added typescript rules for eslint: https://github.com/andrmoel/astronomy-bundle-js/commit/307f4141cd61be98834c643a87f00f2e9b11d3ac